src/test/resources/mssql_tables.sql

```diff
@@ -0,0 +1,13 @@
+create table scheduled_tasks (
+  task_name varchar(250) not null,
+  task_instance varchar(250) not null,
+  task_data  nvarchar(max),
+  execution_time datetimeoffset  not null,
+  picked varchar(10),
```

Comment:
Isn't there a boolean or bit type in mssql?

Reply:
usually is used bit type

Reply:
If it works with `bit` that is better

Reply:
I had not looked at this particular column before. I ran the CompatibilityTest again and the result was 0, so I changed the column to bit and the result as the same. I will submit another commit.